layout/_partial/scripts.ejs

```diff
@@ -66,13 +66,12 @@
 <% } %>
 <!-- Google Analytics -->
 <% if (theme.google_analytics.enabled && theme.google_analytics.id){ %>
-    <script type="text/javascript">
-        (function(i,s,o,g,r,a,m) {i['GoogleAnalyticsObject']=r;i[r]=i[r]||function() {
-        (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-        m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-        })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
-        ga('create', '<%= theme.google_analytics.id %>', 'auto');
-        ga('send', 'pageview');
+    <script async src="https://www.googletagmanager.com/gtag/js?<%= theme.google_analytics.id %>"></script>
```

Comment:
I think the correct URL is `https://www.googletagmanager.com/gtag/js?id=<%= theme.google_analytics.id %>`

Reply:
it doesn't need `id=`.

Reply:
According to the [Google dev guide](https://developers.google.com/analytics/devguides/collection/gtagjs/), it is necessary.

```js
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=GA_MEASUREMENT_ID"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'GA_MEASUREMENT_ID');
</script>
```